Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM fedora:20
 ADD tomprince-hybridlogic-fedora-20-x86_64.repo /etc/yum.repos.d/
 #RUN yum upgrade -y
 RUN yum install -y python-devel python-pip gcc libffi-devel openssl-devel
-RUN pip install buildbot==0.8.9 txgithub boto service_identity
-
+RUN pip install buildbot==0.8.10 txgithub boto service_identity
```

Comment:
It was broken in 0.8.9. The fix is https://github.com/buildbot/buildbot/commit/312f09175ab53aa89f5de60bca837614dcbf5d70 which was in 0.8.10.